### PR TITLE
callToAction8

### PR DESCRIPTION
**Pull request by emreicandan**

### CODE REVIEW

**Review by akhnos — CHANGES_REQUESTED** [review]
1. Native HTML tags should have class so the user can change the default behavior. There should be decorateCss for every element in the component
2. Button action should be handled
3. Unnecessary elements should be removed

File: editor-components/call_to_action/call_to_action8/call_to_action8.tsx [hunk 1]
```diff
@@ -6,150 +6,65 @@ import styles from "./call_to_action8.module.scss";
...
-                <div className={this.decorateCSS("card")}>
-                  <h3 className={this.decorateCSS("rightCardTitle")}>{this.getPropValue("rightCardTitle")}</h3>
-                  <p className={this.decorateCSS("right-card-description")}>{this.getPropValue("rightCardDescription")}</p>
-                  <ComposerLink path={this.getPropValue("rightCardLink")[1].value}>
-                    <span className={this.decorateCSS("button")}>{this.getPropValue("rightCardLink")[0].value}</span>
-                  </ComposerLink>
+          <div className={this.decorateCSS(`content`)}>
+            <div className={this.decorateCSS("title-box")}>
+              <div className={this.decorateCSS("title")}>
+              <h1>{this.getPropValue("title")}</h1>
```
Comment (Ayazelar) [inline editor-components/call_to_action/call_to_action8/call_to_action8.tsx:50]: should have className

File: editor-components/call_to_action/call_to_action8/call_to_action8.tsx [hunk 2]
```diff
@@ -6,150 +6,65 @@ import styles from "./call_to_action8.module.scss";
...
+              </div>
+              <ComposerLink path={this.getPropValue("button_url")}>
+                <div className={this.decorateCSS("button")}>
+                <button>
```
Comment (Ayazelar) [inline editor-components/call_to_action/call_to_action8/call_to_action8.tsx:54]: should have className

File: editor-components/call_to_action/call_to_action8/call_to_action8.tsx [hunk 3]
```diff
@@ -6,150 +6,65 @@ import styles from "./call_to_action8.module.scss";
...
+              {this.getPropValue("button-text")}
+              </button>
                 </div>
+            </ComposerLink>
             </div>
+          {image &&<div className={this.decorateCSS("image-box")}>
+              <img src={this.getPropValue("image")} alt="" />
```
Comment (Ayazelar) [inline editor-components/call_to_action/call_to_action8/call_to_action8.tsx:61]: should have className

File: editor-components/call_to_action/call_to_action8/call_to_action8.tsx [hunk 4]
```diff
@@ -6,150 +6,65 @@ import styles from "./call_to_action8.module.scss";
...
-          key: "buttonlink",
-          displayer: "Button Link",
-          value: "",
-        }
-      ]
+      type: "image",
+      key: "image",
+      displayer: "Image",
+      value:
+        "https://site.sociolib.com/dash/wp-content/uploads/sites/25/2024/04/10.png",
```
Comment (Ayazelar) [inline editor-components/call_to_action/call_to_action8/call_to_action8.tsx:35]: this image must be come from spica

**Review by Ayazelar — APPROVED** [review]

**Pull request merged**